### ISSUE TITLE
Object parent property overrides not displaying

### ISSUE THREAD

**Issue by anaanook** [YellowAfterlife/GMEdit#134]
I'm new to this so maybe I've got something wrong, but here's my issue.

So lets say I've got an object called [parent] and and object [child]. In GMS2 I added property 'flags' to [parent] and it looks like this in the object view of [parent] in GMEdit:

```
#event properties (no comments/etc. here are saved)
parent_index = -1;
uses_physics = false;
flags:real = 0;
```

Now in GMS2 if I override the default value of 'flags' for the inherited property [child].  This is stored in the .yy file but isn't displayed in GMEdit. In the .yy file for [child] the overriden property is stored in a separate region:

```
"overriddenProperties":[
   {
      "propertyId":{
         "name":"flags",
         "path":"objects/parent/parent.yy"
      },
      "objectId":{
         "name":"parent",
         "path":"objects/parent/parent.yy"
      },
      "value":"5",
      "resourceVersion":"1.0",
      "name":"",
      "tags":[],
      "resourceType":"GMOverriddenProperty"
   }
]
```

I think I'd expect overridden properties to be in a separate #event section of the object view in GMEdit? 

Let me know if this is clear, Thanks!

**Comment by YellowAfterlife** [comment]
Indeed, overridden properties are just not implemented as of now since there's a whole lot to them (e.g. you have to recursively traverse parents on load/save/checking for changes) compared to own properties. I plan to look into it at some point.

**Comment by YellowAfterlife** [comment]
Implemented in 4fd9cb82ad3aa6db6d550c0c01d7a1b606ba80a4
